kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -10153,6 +10170,36 @@ public boolean isDropboxMaintainer(String siteId)
 		// (dropbox maintain in their myWorkspace just gives them access to their own dropbox)
 		return m_securityService.unlock(AUTH_DROPBOX_MAINTAIN, m_siteService.siteReference(siteId));
 	}
+	
+	/**
+	 * Determine whether the user has the dropbox.groups permission 
+	 * 
+	 * @return True if user has dropbox.groups permission, false otherwise.
+	 */
+	public boolean isDropboxGroups()
```

Comment:
Generally having a service provide methods that rely on the placement isn't a good idea as it means they don't work when accessed outside the portal ( eg /access or /direct). In your existing code it doesn't look like this method is called so I'd not add it in the first place